src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -214,6 +214,34 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @DELETE
+    @Path("/Own/Shared/:id")
+    public async deleteOwnSharedAttributeAndNotifyPeer(@PathParam("id") attributeId: string): Promise<Envelope> {
+        const result = await this.consumptionServices.attributes.deleteOwnSharedAttributeAndNotifyPeer({ attributeId });
+        return this.ok(result);
+    }
+
+    @DELETE
+    @Path("/Peer/Shared/:id")
+    public async deletePeerSharedAttributeAndNotifyOwner(@PathParam("id") attributeId: string): Promise<Envelope> {
+        const result = await this.consumptionServices.attributes.deletePeerSharedAttributeAndNotifyOwner({ attributeId });
+        return this.ok(result);
+    }
+
+    @DELETE
+    @Path("/ThirdParty/:id")
+    public async deleteThirdPartyAttributeAndNotifyPeer(@PathParam("id") attributeId: string): Promise<Envelope> {
```

Comment:
again naming: deleteThirdPartyOwnedRelationshipAttributeAndNotifyPeer
```suggestion
    public async deleteThirdPartyOwnedRelationshipAttributeAndNotifyPeer(@PathParam("id") attributeId: string): Promise<Envelope> {
```